apps/journeys-admin/src/components/Editor/Toolbar/Items/DetailsItem/DetailsItem.tsx

```diff
@@ -6,7 +6,7 @@ import { setBeaconPageViewed } from '@core/journeys/ui/beaconHooks'
 import { useJourney } from '@core/journeys/ui/JourneyProvider'
 import Edit2Icon from '@core/shared/ui/icons/Edit2'
 
-import { TitleDescriptionDialog } from '../../TitleDescriptionDialog'
+import { JourneyDetailsDialog } from '../../JourneyDetailsDialog'
 import { Item } from '../Item/Item'
```

Comment:
Not related to your current changes, but might as well tidy up while we're here
```suggestion
import { Item } from '../Item'
```